### PR TITLE
Add remove-add-functions

### PR DESCRIPTION
**Pull request by farouqdaif1**

### CODE REVIEW

File: src/uImanpulate.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  editBtn.classList.add('edit');
+  editBtn.setAttribute('id', `edit${task.index}`);
+  editBtn.textContent = 'edit';
+  divman.appendChild(editBtn);
+  // Inside div manipulate create Edit button
+  const deletBtn = document.createElement('span');
+  deletBtn.classList.add('material-icons');
+  deletBtn.classList.add('delete');
+  deletBtn.setAttribute('id', `delete${task.index}`);
+  deletBtn.textContent = 'delete';
```
Comment (EricMbouwe) [inline src/uImanpulate.js]: - [OPTIONAL] I would suggest adding a `cursor: pointer` CSS property to your `edit` and `delete` button to improve the UX of your app.

File: src/index.js [hunk 2]
```diff
@@ -1,79 +1,91 @@
...
+// const theNewInput = document.querySelector('.editing');
+// theNewInput.addEventListener('keypress', (e) => {
+//   if (e.key === 'Enter') {
+//     getDataLS().forEach((el) => {
+//       if (el.description === listItem.textContent) {
+//         console.log('adas');
+//       }
+//     })
+//   }
+// })
```
Comment (EricMbouwe) [inline src/index.js]: - [ ] Let's remove those comments from your codebase since this makes it look messy.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @farouqdaif1,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️
- Styling ✔️

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @farouqdaif1,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/uImanpulate.js (hunk 1 above)
Comment (farouqdaif1) [reply src/uImanpulate.js]: Thank you

File: src/index.js (hunk 2 above)
Comment (farouqdaif1) [reply src/index.js]: Thank you

**Pull request merged**